### DISCUSSION TITLE
What does Operator in the checklist mean?

### DISCUSSION THREAD

**Discussion by sumrise0828** [0x6d69636b/windows_hardening#162] [Q&A] [answered]
Hello, I want to customize a manifest to meet different GCB standards. What do the operators in csv files mean? Or is there a better way to generate a CSV list?
Thanks !

**Comment by 0x6d69636b** [comment, marked as answer]
The operator controls the comparison of the current value with the recommended value. If the setting is to be 1, the operator is =. If the recommendation is 10 or less, the operator is <=. 

In the code you will find all the operators supported by https://github.com/0x6d69636b/windows_hardening/blob/master/HardeningKitty.psm1#L1485.
